packages/gatsby-remark-images/src/index.js

```diff
@@ -81,12 +81,28 @@ module.exports = (
     }
   }
 
-  const getNodeTitle = (node, alt, defaultAlt) => {
-    if (node.title) {
-      return node.title
-    } else if (alt && alt.length > 0 && alt !== defaultAlt) {
-      return alt
+  const getImageCaption = (node, alt, defaultAlt) => {
+    const captionOptions = Array.isArray(options.showCaptions)
+      ? options.showCaptions
+      : options.showCaptions === true
+      ? [`title`, `alt`]
+      : false
+
+    if (captionOptions) {
+      for (let option of captionOptions) {
+        /*eslint-disable */
```

Comment:
@m-allanson eslint didn't like me using fall-throughs in my switch statement here, so I disabled it. Let me know if there is a better way to handle this!

Reply:
Generally you want to avoid blanket disabling eslint so it will still catch other linting errors aside from the ones you are OK with making; explicit is always better :)

```
for (const option of captionOptions) {
  // eslint-disable-next-line no-fallthrough, default-case
  switch (option) {
    // ...
  }
}
```

Reply:
Thank you! I just didn't know the proper syntax for this. I will update the PR :)

Reply:
@superhawk610 updated! thanks for your help!